gemd/units/impl.py

```diff
@@ -9,12 +14,25 @@
 from pint.errors import DimensionalityError as IncompatibleUnitsError  # noqa Import
 from pint.errors import UndefinedUnitError, DefinitionSyntaxError  # noqa Import
 
-import functools
-import pkg_resources
-from typing import Union, List, Tuple, Generator, Any
+# Store directories so they don't get auto-cleaned until exit
+_TEMP_DIRECTORIES = []
+
+
+def _deploy_default_files() -> str:
+    """Copy the units & constants file into a temporary directory."""
+    default_dir = TemporaryDirectory()
+    _TEMP_DIRECTORIES.append(default_dir)
+
+    units_path = Path(default_dir.name) / "citrine_en.txt"
+    units_path.write_text(read_text("gemd.units", "citrine_en.txt"))
```

Comment:
The change in package behavior required a bunch of rework here because the units file imports the constants file via a relative file link, but importlib refuses to give a path-like object (as it may be unpacking a ZIP).